src/bitcoin-cli.cpp

```diff
@@ -151,10 +151,15 @@ static int AppInitRPC(int argc, char* argv[])
             strUsage += FormatParagraph(LicenseInfo());
         } else {
             strUsage += "\n"
-                "Usage:  bitcoin-cli [options] <command> [params]  Send command to " PACKAGE_NAME "\n"
-                "or:     bitcoin-cli [options] -named <command> [name=value]...  Send command to " PACKAGE_NAME " (with named arguments)\n"
-                "or:     bitcoin-cli [options] help                List commands\n"
-                "or:     bitcoin-cli [options] help <command>      Get help for a command\n";
+                "The bitcoin-cli utility provides a command line interface to interact with a " PACKAGE_NAME " RPC server.\n"
```

Comment:
This was better _without_ `PACKAGE_NAME` here (since it isn't tied to a specific program)

Reply:
I also don't agree? This tool is designed to work with Bitcoin Core's RPC server.